css/style.css

```diff
@@ -0,0 +1,582 @@
+
+@media only screen and (min-width: 768px) {
+  .col-tab-1 {
+    width: 8.33%;
+  }
+
+  .col-tab-2 {
+    width: 16.66%;
+  }
+
+  .col-tab-3 {
+    width: 25%;
+  }
+
+  .col-tab-4 {
+    width: 33.33%;
+  }
+
+  .col-tab-5 {
+    width: 41.66%;
+  }
+
+  .col-tab-6 {
+    width: 50%;
+  }
+
+  .col-tab-7 {
+    width: 58.33%;
+  }
+
+  .col-tab-8 {
+    width: 66.66%;
+  }
+
+  .col-tab-9 {
+    width: 75%;
+  }
+
+  .col-tab-10 {
+    width: 83.33%;
+  }
+
+  .col-tab-11 {
+    width: 91.66%;
+  }
+
+  .col-tab-12 {
+    width: 100%;
+  }
+}
+
+/* this is for desktop view  */
+@media only screen and (min-width: 1024px) {
+  .col-desk-1 {
+    width: 8.33%;
+  }
+
+  .col-desk-2 {
+    width: 16.66%;
+  }
+
+  .col-desk-3 {
+    width: 25%;
+  }
+
+  .col-desk-4 {
+    width: 33.33%;
+  }
+
+  .col-desk-5 {
+    width: 41.66%;
+  }
+
+  .col-desk-6 {
+    width: 50%;
+  }
+
+  .col-desk-7 {
+    width: 58.33%;
+  }
+
+  .col-desk-8 {
+    width: 66.66%;
+  }
+
+  .col-desk-9 {
+    width: 75%;
+  }
+
+  .col-desk-10 {
+    width: 83.33%;
+  }
+
+  .col-desk-11 {
+    width: 91.66%;
+  }
+
+  .col-desk-12 {
+    width: 100%;
+  }
+}
+
+/* added media queries for hero area */
+
+@media only screen and (min-width: 768px) {
+  .hero-area {
+    display: grid;
+    grid-template-columns: 66.66% 33.33%;
+    grid-template-rows: auto auto;
+    grid-template-areas: "item1 item2" "item1 item3";
+    grid-gap: 5px;
+    background-color: rgb(43, 226, 89);
+    margin-bottom: 20px;
+  }
+
+  .item1 {
+    grid-area: item1;
+    background-color: darkkhaki;
+  }
+
+  .item2 {
+    grid-area: item2;
+    background-color: rgb(13, 5, 32);
+  }
+
+  .item3 {
+    grid-area: item3;
+    background-color: brown;
+  }
+
+  /* removing the mobile view */
+
+  .mobile-nav {
+    position: absolute;
+    left: -999999px;
+  }
+
+  /* nav-bar for tablet and desktop view */
+  header .upper-nav {
+    background: #131313;
+    display: flex;
+    justify-content: space-between;
+    line-height: 40px;
+    max-height: 40px;
+    width: 100%;
+    color: white;
+    padding: 0 15px 0 110px;
+    font-size: 0.8rem;
+  }
+
+  .nav-bar-left {
+    display: flex;
+    align-items: center;
+    width: 400px;
+    justify-content: space-around;
+  }
+
+  .nav-bar-right {
+    display: flex;
+    align-items: center;
+    width: 200px;
+    justify-content: space-around;
+  }
+
+  .upper-nav ul {
+    list-style: none;
+  }
+
+  .upper-nav ul a {
+    color: rgb(226, 206, 206);
+    text-decoration: none;
+  }
+
+  .upper-nav ul a:hover {
+    color: white;
+    text-decoration: underline;
+  }
+}
+
+/* the lower-nav bar */
+
+@media only screen and (min-width: 768px) {
+  .lower-nav-bar {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
+    z-index: 5;
+  }
+
+  .sticky-nav {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
+    z-index: 5;
+  }
```

Comment:
Group selectors with the same style definition

Reply:
selectors with same style definition grouped